services/nutanix-ai/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+displayName: Nutanix AI
```

Comment:
pls add type: catalog per https://docs.google.com/document/d/1TFvMb8xlEckWDzt4DrVhqENEiwAIefkxTt2loY2LL3Q/edit#bookmark=id.kdal4251v7wo

Reply:
the type has been added.